custom_metrics/media.js

```diff
@@ -93,23 +93,51 @@ return JSON.stringify({
   })(),
   // Count all video nodes
   'num_video_nodes': document.querySelectorAll('video').length,
-  // Returns a set of video node attribute names
-  'video_nodes_attributes': (() => {
-    var allAttributes = getNodesAttributes(document.querySelectorAll('video'));
-    var filter = ['autoplay', 'autoPictureInPicture', 'buffered', 'controls',
+
+  //gets the duration of all videos loaded
+  'video_durations': (() =>{
+    var nodes = document.querySelectorAll('video');
+    var videoDuration = new Set();
+    for(var i=0, len=nodes.length;i<len;i++){
+        var duration = nodes[i].duration;
+        videoDuration.add(duration);
+      
+    }
+    return Array.from(videoDuration);
+})(),
+
+    // Returns a set of video node attribute names, and the count on the page
+    'video_attributes_values_counts': Array.from(document.querySelectorAll('video')).reduce((stats, video) => {
+      const attrs = video.getAttributeNames();
+      var filter = ['autoplay', 'autoPictureInPicture', 'buffered', 'controls',
       'controlslist', 'crossorigin', 'use-credentials', 'currentTime',
       'disablePictureInPicture', 'disableRemotePlayback', 'duration',
       'height', 'intrinsicsize', 'loop', 'muted', 'playsinline', 'poster',
       'preload', 'src', 'width'];
-    return allAttributes.filter(el => filter.includes(el));
-  })(),
-  // Counts the number of pictures using source media min-resolution
-  'num_picture_using_min_resolution': (() => {
```

Comment:
Did you mean to delete this metric?

Reply:
no.  I'll fix.  Thanks for catching.

Reply:
fixes pushed.  I guess I was moving too fast when I deleted that metric.